projects/ad463x_fmcz/Makefile

```diff
@@ -1,3 +1,8 @@
+IIOD=y
+ADAQ4224=n
```

Comment:
So i guess no more than one can be `y` at the same time, in which case a variable like AD463X_ID that can take either one of these 3 values would make more sense. This is not critical but I think it's cleaner.